connection.go

```diff
@@ -110,17 +110,16 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 // msgpack array, cause Tarantool always returns result as an array.
 // For all space related methods and Call* (but not Call17*) methods Tarantool
 // always returns array of array (array of tuples for space related methods).
-// For Eval* and Call17* tarantool always returns array, but does not forces
+// For Eval* and Call17* Tarantool always returns array, but does not forces
 // array of arrays.
-
 type Connection struct {
```

Comment:
http://127.0.0.1:6060/pkg/github.com/tarantool/go-tarantool/#Connection

It seems like doc should have a link here.

![image](https://user-images.githubusercontent.com/20455996/161788615-b6ce4c17-a746-4194-8cf7-d649bc451687.png)

Reply:
It would be really troublesome to implement now, so I've preferred to file an issue for the future.

Reply:
It is not possible to refer to another method using href in gdoc.

Reply:
Yeah, it seems that all links in go doc is autogenerated and there are no links in descriptions, only in code.